postgres/answers/02_explanation.md

```
@@ -0,0 +1,18 @@
+```sql
+SELECT id, name, price
+  FROM products
+ WHERE price < 1000
+ ORDER BY price ASC;
+```
+
+## 解説
+
+- `WHERE price < 1000`により価格が1000未満の商品を抽出しています。
+- `ORDER BY price ASC`で価格の安い順に並べています。
+- 他の句は使用せず、必要な条件と並び順のみを指定しています。
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)